lib/addExcursionPointers.m

```diff
@@ -0,0 +1,132 @@
+%==========================================================================
+%This function takes in an NIDM-Results json and creates a hashmap labelling
+%objects with a pointer to their relevant excursion set. The resultant
+%hashmap takes in the id of an object from the main NIDM-Results graph and
+%outputs a number for it's corresponding ExcursionMap.
```

Comment:
Could you provide a bit more details about how this hashmap is actually implemented? I can see that you have two variable `labelValues` and `labelKeys` can you state explicitely what each contains. A short example would be great.